actions/mysql_show_create_table_action_test.go

```diff
@@ -0,0 +1,72 @@
+// pmm-agent
+// Copyright (C) 2018 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package actions
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/percona/pmm/api/agentpb"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/percona/pmm-agent/utils/tests"
+)
+
+func TestShowCreateTable(t *testing.T) {
+	db := tests.OpenTestMySQL(t)
+	defer db.Close() //nolint:errcheck
+	tests.MySQLVersion(t, db)
+
+	t.Run("Default", func(t *testing.T) {
+		t.Parallel()
+
+		params := &agentpb.StartActionRequest_MySQLShowCreateTableParams{
+			Dsn:   "root:root-password@tcp(127.0.0.1:3306)/world",
+			Table: "city",
+		}
+		a := NewMySQLShowCreateTableAction("", params)
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+		defer cancel()
+
+		b, err := a.Run(ctx)
+		require.NoError(t, err)
+
+		expected := strings.TrimSpace("CREATE TABLE `city`")
+
+		actual := strings.TrimSpace(string(b))
+		assert.Regexp(t, expected, actual)
```

Comment:
why assert.Regexp? Maybe it's better to use `https://godoc.org/github.com/stretchr/testify/assert#Contains`?

Reply:
And here too https://github.com/percona/pmm-agent/pull/67/files#diff-014faa1e76a7479251aee8837066b8a2R55